extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,72 +1,74 @@
-						'wds-icon wds-icon-small wds-global-navigation__search-label-icon'
-					) ?>
-					<input class="wds-global-navigation__search-input"
-						name="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['param-name'] ); ?>"
-						data-active-placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-active'] ); ?>"
-						placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
-				</label>
-				<button class="wds-button wds-is-text wds-global-navigation__search-close" type="reset">
-					<?= DesignSystemHelper::getSvg(
-						'wds-icons-cross',
-						'wds-icon wds-icon-small wds-global-navigation__search-close-icon',
-						wfMessage( 'global-navigation-search-cancel' )->escaped()
-					) ?>
-				</button>
-			</div>
-			<button class="wds-button wds-global-navigation__search-submit">
-				<?= DesignSystemHelper::getSvg(
-					'wds-icons-arrow',
-					'wds-icon wds-icon-small wds-global-navigation__search-submit-icon'
-				) ?>
-			</button>
-		</form>
-		<?php if ( isset( $model['user'] ) ): ?>
+<div class="wds-global-navigation-wrapper">
+	<div id="globalNavigation" class="wds-global-navigation <?= isset( $model['fandom_overview'] ) ? '' : ' wds-search-is-always-visible'; ?>">
+		<div class="wds-global-navigation__content-bar">
+			<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'logo', [ 'model' => $model['logo'] ] ); ?>
+			<?php
+			if ( isset( $model['fandom_overview'] ) ):
+				foreach ( $model['fandom_overview']['links'] as $link ):
+			?>
+					<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkBranded', [ 'model' => $link ] ); ?>
+			<?php
+				endforeach;
+			endif;
+			?>
 			<?= $app->renderView(
 				'DesignSystemGlobalNavigationService',
 				'links',
 				[
-					'model' => $model['user'],
-					'type' => 'user-menu',
-					'dropdownRightAligned' => true,
+					'model' => $model['wikis'],
+					'type' => 'wikis-menu'
 				]
 			); ?>
-			<?= $app->renderView( 'DesignSystemGlobalNavigationWallNotificationsService', 'index' ); ?>
-		<?php elseif ( isset( $model['anon'] ) ): ?>
-			<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'accountNavigation', [ 'model' => $model['anon'] ] ); ?>
-		<?php endif; ?>
-		<div class="wds-global-navigation__start-a-wiki <?= isset( $model['user'] ) ? ' wds-user-is-logged-in' : ''; ?>">
-			<a href="<?= Sanitizer::encodeAttribute( $model['create_wiki']['header']['href'] ); ?>" class="wds-button wds-is-squished wds-is-secondary">
-				<span class="wds-global-navigation__start-a-wiki-caption"><?= DesignSystemHelper::renderText( $model['create_wiki']['header']['title'] ) ?></span>
-				<?= DesignSystemHelper::getSvg(
-					'wds-icons-plus',
-					'wds-global-navigation__start-a-wiki-icon wds-icon'
-				) ?>
-			</a>
+			<form class="wds-global-navigation__search" action="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['url'] ); ?>">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "search" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "module" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)